tests/core/tests/test_resources/test_modelresource/test_data_deletion_dry_run.py

```diff
@@ -0,0 +1,155 @@
+from unittest import mock
+
+import tablib
+from core.models import Book
+from core.tests.resources import BookResource
+from core.tests.utils import ignore_widget_deprecation_warning
+from django.test import TestCase
+
+from import_export import fields, results, widgets
+
+
+class DataDeletionDryRunTests(TestCase):
```

Comment:
Please can we keep the convention of using "Test" not "Tests".  For example, I prefer that this name is `DataDeletionDryRunTest`.  Please can you modify other test cases to end with `Test` and not `Tests`